test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo88 = '' ;
+
+if(foo88  === "b88") {
+  console.log( 'he88' );
+}
+
+
+var foo77 = '' ;
+
+if(foo77  === "baaaaaaaaaar") {
```

Comment:
Codacy has a fix for the issue: [Replace `(foo77·` with `·(foo77`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
if (foo77 === "baaaaaaaaaar") {
```